activestorage/lib/active_storage/engine.rb

```diff
@@ -18,6 +18,14 @@ class Engine < Rails::Engine # :nodoc:
     config.active_storage.analyzers = [ ActiveStorage::Analyzer::ImageAnalyzer, ActiveStorage::Analyzer::VideoAnalyzer ]
     config.active_storage.paths = ActiveSupport::OrderedOptions.new
     config.active_storage.variable_content_types = [ "image/png", "image/gif", "image/jpg", "image/jpeg", "image/vnd.adobe.photoshop" ]
+    config.active_storage.content_types_to_render_as_binary = [ "text/html",
```

Comment:
Perhaps a better name for this would e `content_types_to_render_as_attachment`.

Reply:
I’m fine with either `binary` or `attachment`, but `render` doesn’t seem like the right word to me. How about `content_types_to_serve_as_binary`?